.gitignore

```
@@ -1,3 +1,4 @@
 node_modules/
 tsconfig.example.json
-dist/
\ No newline at end of file
+dist/
+.env
```

```suggestion
.env
coverage
```
커버리지 결과 파일도 지금보니 ignore 시켜야겠네요

---

ignore하는걸로 추가하겠습니다~!